README.md

```diff
@@ -31,8 +31,22 @@ You can create segments for html blogs, documentations, repositories, books and
 </summary>
 <ul>
     <li><a href="https://html5doctor.com"> HTML5 Doctor</a></li>
+    <li><a href="https://progate.com/courses/html">Progate</a></li>
     <li><a href="https://html5rocks.com"> HTML5 Rocks</a></li>
```

Comment:
HTML5 Rocks is now **Web Fundamentals by Google Developers**
New link: https://developers.google.com/web